src/main/java/com/odysseusinc/arachne/portal/model/ParticipantRole.java

```diff
@@ -88,7 +91,8 @@
             DELETE_DATASOURCE,
             ACCESS_DATASOURCE,
             UNLINK_DATASOURCE,
-            ACCESS_PAPER
+            ACCESS_PAPER,
+            EDIT_ANALYSIS
```

Comment:
Where is usage of this perm in real edit actions? (edit analysis title, descr, etc)?